internal/fs/wrappers/monitoring_test.go

```diff
@@ -117,3 +124,148 @@ func TestFsErrStrAndCategory(t *testing.T) {
+	return ex
+}
+
+type dummyFS struct{}
+
+func (d dummyFS) StatFS(_ context.Context, _ *fuseops.StatFSOp) error {
+	return nil
+}
+
+func (d dummyFS) LookUpInode(_ context.Context, _ *fuseops.LookUpInodeOp) error {
+	return nil
+}
+
+func (d dummyFS) GetInodeAttributes(_ context.Context, _ *fuseops.GetInodeAttributesOp) error {
+	return nil
+}
+
+func (d dummyFS) SetInodeAttributes(_ context.Context, _ *fuseops.SetInodeAttributesOp) error {
+	return nil
+}
+
+func (d dummyFS) ForgetInode(_ context.Context, _ *fuseops.ForgetInodeOp) error {
+	return nil
+}
+
+func (d dummyFS) BatchForget(_ context.Context, _ *fuseops.BatchForgetOp) error {
+	return nil
+}
+
+func (d dummyFS) MkDir(_ context.Context, _ *fuseops.MkDirOp) error {
+	return nil
+}
+
+func (d dummyFS) MkNode(_ context.Context, _ *fuseops.MkNodeOp) error {
+	return nil
+}
+
+func (d dummyFS) CreateFile(_ context.Context, _ *fuseops.CreateFileOp) error {
+	return nil
+}
+
+func (d dummyFS) CreateLink(_ context.Context, _ *fuseops.CreateLinkOp) error {
+	return nil
+}
+
+func (d dummyFS) CreateSymlink(_ context.Context, _ *fuseops.CreateSymlinkOp) error {
+	return nil
+}
+
+func (d dummyFS) Rename(_ context.Context, _ *fuseops.RenameOp) error {
+	return nil
+}
+
+func (d dummyFS) RmDir(_ context.Context, _ *fuseops.RmDirOp) error {
+	return nil
+}
+
+func (d dummyFS) Unlink(_ context.Context, _ *fuseops.UnlinkOp) error {
+	return nil
+}
+
+func (d dummyFS) OpenDir(_ context.Context, _ *fuseops.OpenDirOp) error {
+	return nil
+}
+
+func (d dummyFS) ReadDir(_ context.Context, _ *fuseops.ReadDirOp) error {
+	return nil
+}
+
+func (d dummyFS) ReleaseDirHandle(_ context.Context, _ *fuseops.ReleaseDirHandleOp) error {
+	return nil
+}
+
+func (d dummyFS) OpenFile(_ context.Context, _ *fuseops.OpenFileOp) error {
+	return nil
+}
+
+func (d dummyFS) ReadFile(_ context.Context, _ *fuseops.ReadFileOp) error {
+	return nil
+}
+
+func (d dummyFS) WriteFile(_ context.Context, _ *fuseops.WriteFileOp) error {
+	return nil
+}
+
+func (d dummyFS) SyncFile(_ context.Context, _ *fuseops.SyncFileOp) error {
+	return nil
+}
+
+func (d dummyFS) FlushFile(_ context.Context, _ *fuseops.FlushFileOp) error {
+	return nil
+}
+
+func (d dummyFS) ReleaseFileHandle(_ context.Context, _ *fuseops.ReleaseFileHandleOp) error {
+	return nil
+}
+
+func (d dummyFS) ReadSymlink(_ context.Context, _ *fuseops.ReadSymlinkOp) error {
+	return nil
+}
+
+func (d dummyFS) RemoveXattr(_ context.Context, _ *fuseops.RemoveXattrOp) error {
+	return nil
+}
+
+func (d dummyFS) GetXattr(_ context.Context, _ *fuseops.GetXattrOp) error {
+	return nil
+}
+
+func (d dummyFS) ListXattr(_ context.Context, _ *fuseops.ListXattrOp) error {
+	return nil
+}
+
+func (d dummyFS) SetXattr(_ context.Context, _ *fuseops.SetXattrOp) error {
+	return nil
+}
+
+func (d dummyFS) Fallocate(_ context.Context, _ *fuseops.FallocateOp) error {
+	return nil
+}
+
+func (d dummyFS) Destroy() {}
+
+func TestSpan(t *testing.T) {
```

Comment:
The name is a bit unclear to me. This sounds like it's testing the span functionality, which is not a gcsfuse feature.. How about TestInvocationSpan ?